portfolio/src/main/webapp/project.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Project</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div class="nav" id="mynav">
+		<a href="index.html">Home</a>
+		<a class="active" href="project.html">Projects</a>
+		<a href="experience.html">Experiences</a>
+		<a href="#" class="icon" onclick="myFunction()">&#9776;</a>
+	</div>
+	<div class="content">
+		<h4><a href="https://github.com/keyuyan1145/RedBubbleProject">Database System for E-Commerce</a></h4>
+		<p><i>Spring 2020</i></p>
+		<p>
+			In a team of two, we implemented a database system on 
+			Google Cloud for an e-commerce, Red Bubble, with PostgreSQL 
+			as a class project. 
+			Then, we programmed a functional website with Flask to interact with the database,
+			such as displaying products in the catalog, verifying user logins, 
+			and posting product reviews. 
+		</p>
+		<h4><a href="https://git.io/Jf98M">Web Server with TCP Protocol</a></h4>
+		<p><i>Spring 2020</i></p>
+		<p>
+			I programmed a web server from scratch in C that allows 
+			up to 5 client connections. 
```

Comment:
Purely curiosity on my part and not a review comment: why only 5? With `select` or pthreads?

Reply:
It's actually an interesting question, but the answer might be disappointing. There's not even a clear reason for the number 5. The project spec didn't put an upperbound on the number of connections, and I took this number as a reasonable limit from the lecture example.

Reply:
Smile. Most code has the proof-by-induction property: if you can do 0, 1, n, n+1 you can do all of the n for non-ridiculous values of n and the huge implementation work item is usually going from 1 to a value greater than 1.